Sources/SourceKit/SourceKitServer.swift

```diff
@@ -269,6 +270,9 @@ extension SourceKitServer {
         clientCapabilities: req.params.capabilities.textDocument?.codeAction,
         codeActionOptions: CodeActionOptions(codeActionKinds: nil),
         supportsCodeActions: false // TODO: Turn it on after a provider is implemented.
+      ),
+      executeCommandProvider: ExecuteCommandOptions(
+        commands: builtinSwiftCommands // FIXME: Clangd commands?
```

Comment:
You can make any steps to get closer to opening it still learning

Reply:
Hmm, that's unfortunate.  We load clangd dynamically through the toolchain, so there's no good way for us to get the response from clang this early.  Maybe someday this would be a good place for the specification to evolve to add dynamic capabilities for the server (there are already dynamic capabilities for the *client*). As a short term hack, we could maintain a list of every command clangd supports, but that doesn't need to be done in this patch.

For now, this is fine.